initializers/active_model_serializer.rb

```diff
@@ -0,0 +1 @@
+ActiveModel::ArraySerializer.root = false
```

Comment:
@isratrade we ran into an issue where when active model serializers is loaded into Satellite it alters the "render :json" behavior of some Satellite routes.

Namely the /rhsm/ route which is served by candlepin_proxies controller.

/rhsm/ essentially proxies requests to the candlepin API, then returns the responses.

when active model serializers is present in the application, it causes the returned response from 'candlepin_proxies' to be wrapped in

{"candlepin_proxies":  ORIGINAL_RESPONSE}

@jmontleon saw the issue is related to different behavior for ArraySerializer, so the workaround proposed here is to restore the functionality for regular "render :json" for rest of Satellite app.....then in our foretello code we override with the behavior active model serializers expects.

so default initializer is ArraySerializer.root = false